### PR TITLE
Added support for auto-updated resource managers

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: ecc7a45dab95fb8e588fe00e5631e63f5c9bec3c
  * Your commit message body contains a line that is longer than 80 characters

Guidelines are available at https://github.com/zhmcclient/python-zhmcclient/blob/master/CONTRIBUTING.rst

---
This message was auto-generated by https://gitcop.com

File: examples/show_auto_updated_partition_manager.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+# Copyright 2022 IBM Corp. All Rights Reserved.
```
Comment (simon-spinner) [inline examples/show_auto_updated_partition_manager.py:2]: We now have 2023

File: examples/show_auto_updated_partition_manager.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+mode.
+
+The shorter durations of list() indicate that the locally maintained list was
+used; the longer durations indicate that a list operation was performed on the
+HMC.
+"""
+
+import sys
+import uuid
+# import logging
```
Comment (simon-spinner) [inline examples/show_auto_updated_partition_manager.py:27]: Why is this import commented out?

File: examples/show_auto_updated_partition_manager.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    part_name = "zhmc_test_{}".format(uuid.uuid4())
+    print("Creating partition {} ...".format(part_name))
+    part_props = {
+        'name': part_name,
+        'description': 'Original partition description.',
+        'cp-processors': 2,
+        'initial-memory': 1024,
+        'maximum-memory': 2048,
+        'processor-mode': 'shared',
+        'boot-device': 'test-operating-system'
```
Comment (simon-spinner) [inline examples/show_auto_updated_partition_manager.py:112]: We should not use 'test-operating-system' - it is not documented and not available on all machines.

Comment (andy-maier) [reply examples/show_auto_updated_partition_manager.py:117]: This was just a rename of the script, but ok so let's fix it. I have removed the boot-device property as it is not needed to the purpose of showing property updates.

**Pull request merged**